docs/source/models/vlm.rst

```diff
@@ -49,7 +48,7 @@ To initialize a VLM, the aforementioned arguments must be passed to the ``LLM``
 To pass an image to the model, note the following in :class:`vllm.inputs.PromptStrictInputs`:
 
 * ``prompt``: The prompt should have a number of ``<image>`` tokens equal to ``image_feature_size``.
-* ``multi_modal_data``: This should be an instance of :class:`~vllm.multimodal.image.ImagePixelData` or :class:`~vllm.multimodal.image.ImageFeatureData`.
+* ``multi_modal_data``: This is a loosely structured dict that contains multi modal data. 
```

Comment:
Mention the expected schema (see my other comment in LLMInputs).

Reply:
See my other comments, I prefer this to just be a dict and not overload.